extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -191,10 +236,27 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper'], function (help
 				duplicate.attr('checked', false);
 				hiddenDuplicate = duplicate.parent().hide();
 			}
+			$descWikiWrapper.find('label input[type="checkbox"]').change(onCategorySelection);
 		}
 	}
 
+	function onCategorySelection() {
+		track({
+			label: 'category-select'
+		});
+	}
+
+	function onIntendedForKidsCheckboxChange() {
+		track({
+			label: 'intended-for-kids'
```

Comment:
Trevor: "It is enough to track only event of interacting with given options, no matter which one is chosen.
We'll get browser lang and user lang via Google Analytics, and can export wiki lang data (assuming it is created.) I highly doubt we'd change the CNW funnel for a specific language."